commit 09372f3cbeed0a6a8a3ea3fafeb7e699984bee80
Author: Adam Ford <aford173@gmail.com>
Date:   Sun Oct 28 15:28:32 2018 -0500

    ARM: dts: LogicPD Torpedo: Fix mmc3_dat1 interrupt
    
    [ Upstream commit 6809564d64ff1301d44c13334cc0e8369e825a20 ]
    
    When the Torpedo was first introduced back at Kernel 4.2,
    the interrupt extended flag has been set incorrectly.
    
    It was subsequently moved, so this patch corrects Kernel
    4.18+
    
    Fixes: a38867305203 ("ARM: dts: Move move WiFi bindings to
    logicpd-torpedo-37xx-devkit") # v4.18+
    
    Signed-off-by: Adam Ford <aford173@gmail.com>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/logicpd-torpedo-37xx-devkit.dts b/arch/arm/boot/dts/logicpd-torpedo-37xx-devkit.dts
index 9d5d53fbe9c0..c39cf2ca54da 100644
--- a/arch/arm/boot/dts/logicpd-torpedo-37xx-devkit.dts
+++ b/arch/arm/boot/dts/logicpd-torpedo-37xx-devkit.dts
@@ -35,7 +35,7 @@
  * jumpering combinations for the long run.
  */
 &mmc3 {
-	interrupts-extended = <&intc 94 &omap3_pmx_core2 0x46>;
+	interrupts-extended = <&intc 94 &omap3_pmx_core 0x136>;
 	pinctrl-0 = <&mmc3_pins &mmc3_core2_pins>;
 	pinctrl-names = "default";
 	vmmc-supply = <&wl12xx_vmmc>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1027_09372f3cbeed0a6a8a3ea3fafeb7e699984bee80.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1027_09372f3cbeed0a6a8a3ea3fafeb7e699984bee80.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
